### ISSUE TITLE
Remove Advertisements feature shouldn't be on by default

### ISSUE THREAD

**Issue by rea987** [YePpHa/YouTubeCenter#2304]
Hi,

Most of the ads are really annoying. But content creators and legitimate content owners shouldn't be punished for publishing their material for free on YouTube. It is nice to have Remove Advertisements feature but it should be the user who decide whether use it or not. Besides, I am pretty sure the majority of users of that extension/script have already AdBlock installed on their browser.

Thanks.

**Comment by Yonezpt** [comment]
> Besides, I am pretty sure the majority of users of that extension/script have already AdBlock installed on their browser.

Then, according to your certainty, making the feature on or off by default will not make a difference, which makes this a moot point.

**Comment by ErdeFB** [comment]
In my opinion, this shouldn't even be in the feature pool of YTC since ad blocking is better handled with dedicated addons. I'm personally one of those people who block ads on youtube with the exception of people I subscribe to. YTC doesn't allow the fine tuning of where ads are shown whilst an addon like unlock does.

Then again, as this addon is no longer under active development having this discussion is rather pointless.

**Comment by gopego** [comment]
I agree, I disabled few channels on my adblock but I still can't see the ads but then I remember YC disabled ads, I wish we can disabled it, if some people can tell us how to disabled it in code that would be great, thx

**Comment by sangohan** [comment]
or have an opt-in list for channels you'd like to support (but can't donate real money to). I dislike ads, as i prefer that people pay for content, but sometimes practicality and reality is harsh, and for many channels, their audience isn't really in a position to pay real money.
